### ISSUE TITLE
Login php traccar original

### ISSUE THREAD

**Issue by conde000011100** [gpsguru/traccar-ui-php#4]
How about, what would be the way to be able to put the login with the PHP API and when clicking on the start session, open us directly in the session?

**Comment by jaimzj** [comment]
I do not understand your question, where do you click on "start session" ?? and what exactly are you trying to ask? please explain clearly.

**Comment by conde000011100** [comment]
That is to say, personalize the login and when clicking on start session between direct to traccar, sorry for the bad explanation

**Comment by jaimzj** [comment]
where is "start session" that you are trying to click?

what do you mean by start session between direct to traccar? what is direct?

I still do not understand the question, what is your question? or what are you trying to achieve?

**Comment by conde000011100** [comment]
I want to login in PHP, practically change the login nothing more

**Comment by conde000011100** [comment]
I'm giving an example of how it would be what I try to do or want to do.

1.- Login session login with personalized login
login https://subefotos.com/ver/?29c20e05ae7e6dee725f4dbc97ae3598o.png#codigos

2.- Go directly to the traccar system
sistem https://subefotos.com/ver/?3510d752e265ef3e0e013e3a4e4463b8o.png#codigos

regards

**Comment by jaimzj** [comment]
There is no point in hosting a login page elsewhere in such case, what you can do is modify traccars login page and cutomize its UI/design .

**Comment by conde000011100** [comment]
I understand, you have news about the other functions of Api PHP, when they will be ready, do you even have plans for this project?

**Comment by jaimzj** [comment]
which other functions?

Plans are clearly defined in our readme files.

**Comment by conde000011100** [comment]
I refer to releases, master updates

**Comment by jaimzj** [comment]
Yes there will be releases. and master updates! you may follow them.

**Comment by conde000011100** [comment]
Very good regards
